src/components/app/routes/createAppRouter.jsx

```diff
@@ -75,6 +75,23 @@ export default function createAppRouter(queryClient) {
               />
             )}
           />
+          <Route
+            path="skills-quiz"
+            lazy={async () => {
+              const { makeSearchLoader } = await import('../../search');
+              const { SkillsQuizPage } = await import('../../skills-quiz');
+              return {
+                Component: SkillsQuizPage,
+                loader: makeSearchLoader(queryClient),
```

Comment:
[clarification] Does `SkillsQuizPage` use any queries from `makeSearchLoader`? The search loader is responsible for seeding the query cache for the academies list and highlight sets list query. I don't seem to see these calls made on the `skills-quiz` route in stage/production? If `SkillsQuizPage` is not dependent on the data seeded in `makeSearchLoader`, should the `loader` be removed for this route?

Reply:
Good callout, removing the loader now. Opted not to refactor the APIs in this path for scope reasons but will capture those APIs and opportunities in a separate ticket 👍🏽